### PR TITLE
Create a QUESTIONS.md file for mentors

### PR DESCRIPTION
**Pull request by MynahMarie**
These questions are for mentors to use as a guide in order to help futre students solidify their understanding of testing and TDD as well as to try and sensitize them to the importance of testing.

Relates #1

### CODE REVIEW

**Comment by zurda** [comment]
@MynahMarie I've reviewed this, and it looks great. Do you happen to know who can approve this? We're preparing week 2 for FAC-16, and I'd love to get this merged

**Comment by oliverjam** [comment]
@zurda I can approve it. I've got a couple of typos/phrasing changes to the PR—do you want to make the edits? If it's easier we can merge and then update after

File: QUESTIONS.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Reference: [What Are Pure Functions And Why Use Them?](https://medium.com/@jamesjefferyuk/javascript-what-are-pure-functions-4d4d5392d49c)
+
+
+### When using `tape`, what's the difference between 'equal', 'deepEqual' and 'ok'?
+
+Reference: [Tape - Guide](https://ci.testling.com/guide/tape)
+
+
+### How do we call the process of testing small chunks of code?
```
Comment (oliverjam) [inline QUESTIONS.md:23]:
```suggestion
### What do we call the process of testing small chunks of code?
```

File: QUESTIONS.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+### What are some other popular testing frameworks for Javascript and Node?
+
+Jasmine, Mocha, Jest, ...
+Reference: [Comparing Jasmine, Mocha, AVA, Tape and Jest](https://raygun.com/blog/javascript-unit-testing-frameworks)
+
+
+### What are some known advantages of using TDD? 
+### How does using TDD can help develop an application that is more secure?
```
Comment (oliverjam) [inline QUESTIONS.md:35]:
```suggestion
### How can using TDD help develop an application that is more secure?
```

**Review by oliverjam — CHANGES_REQUESTED** [review]

**Comment by zurda** [comment]
This is a duplicate of: https://github.com/foundersandcoders/testing-tdd-intro/pull/3
Closing this PR since the changes are fixed in the PR linked above

**Pull request closed without merging**